cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -791,7 +799,16 @@ public CloudQuota getQuota(String quotaName, boolean required) {
 
     @Override
     public List<ApplicationLog> getRecentLogs(String applicationName) {
```

Comment:
I think that we should add getRecentLogs(String guid) so it can be used to save one additional call if we already have the guid

Reply:
To the public interface?
Can we edit that?
And the interface for the application logs has always been with app name. 
I agree that it would be beneficial (for e.g. performance) to not need a call for getting the guid if we have it but I think that complying with the interface is better.

Reply:
It is ours...
You can add it to CCRC and CCRCImlp and then to CCC and CCCImpl